packages/flutter/lib/src/widgets/widget_inspector.dart

```diff
@@ -868,8 +868,17 @@ mixin WidgetInspectorService {
     registerServiceExtension(
```

Comment:
add a test to widget_inspector_test as well.

Reply:
done.